activerecord/test/cases/encryption/extended_deterministic_queries_test.rb

```diff
@@ -9,19 +9,19 @@ class ActiveRecord::Encryption::ExtendedDeterministicQueriesTest < ActiveRecord:
   end
 
   test "Finds records when data is unencrypted" do
-    ActiveRecord::Encryption.without_encryption { UnencryptedBook.create! name: "Dune" }
+    UnencryptedBook.create!(name: "Dune").tap { |b| assert_equal "Dune", b.attributes_before_type_cast["name"] }
```

Comment:
You can use the existing `#assert_encrypted_attribute` and `assert_not_encrypted_attribute` helpers already available:

```suggestion
UnencryptedBook.create!(name: "Dune").tap { |book| assert_encrypted_attribute book, :name, "Dune" }
```

You are right that the `.without_encryption` context was redundant in the test. I think we can remove the `.tap` assertions in both tests, as we are just testing that the declaration works.

Reply:
Oh perfect. THanks.

Reply:
I'll just remove the `tap` part, I mostly put it there to get your attention and confirm it was behaving as you intended when you wrote the tests.